### ISSUE TITLE
Dock not appearing when using absolute tracking

### ISSUE THREAD

**Issue by Axilotl17** [9tin9tin9/Mac-trackpad-mapper#4]
When using absolute tracking, the Dock at the bottom will not appear when the cursor is moved to the bottom of the screen. however, the Menu bar at the top does appear when moving to the top of the screen. 
**The dock also works fine when positioned vertically at the sides of the screen.**

```C
        fingerPosition = map(f->normalizedVector.position.x, 1 - f->normalizedVector.position.y);
```
I thought it might be the fact apple has the trackpad position inverted on the y (fixed by line 36 in `trackpad_mapper_util.c`). However, even when it is inverted by removing the `1 - `, the dock refuses to appear.

**Comment by Axilotl17** [comment]
Just realized this is because of b17c15e00c099130aab76f04d49e2d3d29ff1d86; but there has to be a way to fix both issues.

**Comment by lokxii** [comment]
Actually I don't understand what information is missing when generating mouse events that caused the dock not to show. Gonna investigate it.

**Comment by lokxii** [comment]
Ok, so for the dock to show up, the cursor has to be one pixel away from the boundaries. When dock is at bottom, `cursor.y` has to be `screenHeight - 1`, when dock is on the left, `cursor.x` has to be `0 + 1`. I think we should handle this in `trackpad_mapper_util.c` but not `settings.h`

**Comment by Axilotl17** [comment]
> When dock is at bottom, `cursor.y` has to be `screenHeight - 1`, when dock is on the left, `cursor.x` has to be `0 + 1`

Why do you think it runs fine at the side but not the bottom?

**Comment by lokxii** [comment]
The side you mean both right and left? Actually idk why is that

**Comment by lokxii** [comment]
Problem solved in eeb144bf7d7fa2271acdb551feefc037d6eaba72. Closing the issue now.
